sky/provision/kubernetes/utils.py

```diff
@@ -60,6 +65,21 @@
 PORT_FORWARD_PROXY_CMD_TEMPLATE = 'kubernetes-port-forward-proxy-command.sh'
 PORT_FORWARD_PROXY_CMD_PATH = '~/.sky/kubernetes-port-forward-proxy-command.sh'
 
+POD_STATUSES = {
+    'Pending', 'Running', 'Succeeded', 'Failed', 'Unknown', 'Terminating'
+}
+AUTODOWN_ANNOTATIONS_KEY = 'skypilot.co/autodown'
+IDLE_MINUTES_TO_AUTOSTOP_ANNOTATIONS_KEY = (
+    'skypilot.co/idle_minutes_to_autostop')
+KUBERNETES_AUTODOWN_ANNOTATIONS_KEYS = [
+    AUTODOWN_ANNOTATIONS_KEY, IDLE_MINUTES_TO_AUTOSTOP_ANNOTATIONS_KEY
+]
+ANNOTATIONS_POD_NOT_FOUND_ERROR_MSG = ('Pod {pod_name} not found in namespace '
+                                       '{namespace} while trying to {action} '
+                                       'Annotations {annotations}.')
+
+V1Pod = Any
```

Comment:
I think we should generally avoid this pattern since it can shadow if V1Pod is imported in the future. Maybe use `Any` directly below?